src/strings/PowerFxResources.en-US.resx

```diff
@@ -2883,15 +2887,15 @@
     <value>Language code of the supplied text.</value>
   </data>
   <data name="AboutTable" xml:space="preserve">
-    <value>Creates a table from the specified records, with as many columns as there are unique record fields. For example: Table({key1: val1, key2: val2, ...}, ...)</value>
+    <value>Creates a table from the specified records and tables, with as many columns as there are unique record fields. For example: Table({key1: val1, key2: val2, ...}, ...)</value>
     <comment>Description of 'Table' function.</comment>
   </data>
   <data name="TableArg1" xml:space="preserve">
-    <value>record</value>
-    <comment>function_parameter - Argument of the Table function - a record that will become a row in the resulting table.</comment>
+    <value>record/table</value>
+    <comment>function_parameter - Argument of the Table function - a record/table that will be part of the the resulting table.</comment>
```

Comment:
>. [](http://example.com/codeflow?start=126&length=1)

Can you add this extra sentence to the comment: `Translate this string. Maintain as a single word (do not add spaces).` We've had issues in the past and this helped prevent those.